layouts/default.vue

```diff
@@ -89,6 +88,15 @@ export default {
         });
       }
 
+      const isCluster = product.inStore === 'cluster';
+
+      if (isCluster) {
```

Comment:
This is the part that needs to be gated on whether the cluster user has permissions to create, edit or delete the `ui.cattle.io.navlink` type (in theory they may not be able to create but they could edit, same kind of thing with delete). All are available by the schema (edit/delete in the schema are a bit odd, but for this purpose we'll use them).

```
const schema = this.$store.getters[`cluster/schemaFor`](UI.NAV_LINK) // User may not have GET permissions so this might be null
const canCreate =  schema?.collectionMethods.find(verb => verb === 'POST')
const canEdit =  schema?.resourceMethods.find(verb => verb === 'PUT')
```

You'll also want to gate this feature on the current product being `EXPLORER` and the isCluster

Reply:
Since we cannot identify the mode for this view, I'm allowing if one of them is available: `if (isClusster && (canCreate || canEdit))`